flavoUr/settings.py

```diff
@@ -55,10 +56,12 @@
 
 ROOT_URLCONF = 'flavoUr.urls'
 
+PROJECT_PATH = os.path.realpath(os.path.dirname(__file__))
+
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(PROJECT_PATH, 'templates')],
```

Comment:
ok i looked over this whole thing and it's AWESOME. this is the one line where i'm not sure what it's doing. i'm sure you're right about it - i just wanna make sure i understand it before we merge. we can go over tomorrow?